### DISCUSSION TITLE
Question about RaiSim solver

### DISCUSSION THREAD

**Discussion by GichanCho** [raisimTech/raisimLib#190] [Q&A]
Hello, this is a student of Motion Control Lab, DGIST.

Nowadays I try to dynamics verification of RaiSim. In the process, I compare data(joint angle, angular velocity,...) from RaiSim and data from Simscape MATLAB.

The trends of the data were generally similar, but there were some differences.

It seems like the problem is due to the difference in solvers, but I'm curious what you're using as RaiSim's solver.

**Comment by jhwangbo** [comment]
https://ieeexplore.ieee.org/abstract/document/8255551?casa_token=CrNaonXgM7QAAAAA:dX1E4FPEk3KdUTND8EazOVAjaNQNEkIX2z1wV4EFesygcY9dK9Hq30j0IzgH-9PPfzEDuahz-g

**Comment by jhwangbo** [comment]
Check this link as well. 

https://github.com/leggedrobotics/SimBenchmark

**Reply by GichanCho** [reply]
Thank you!! It was really helpful.
